cobc/typeck.c

```diff
@@ -3588,6 +3588,7 @@ check_argument_conformance (struct cb_program *program, cb_tree argument_tripple
 	} else {
 		arg_field = NULL;
 	}
+	if (!CB_FIELD_P(param)) return;
```

Comment:
That's unrelated but important, I guess there's also  test that can (and should be) added for this.

Reply:
Not sure how to trigger this one.

Reply:
Possibly passing a constant/literal? I'd need to check the context.

Reply:
Let's move this and its Changelog entry to a separate PR and we can work out the right test (if @lefessan does not remember).